ios/Runner/Info.plist

```diff
@@ -101,5 +101,15 @@
 	<false/>
 	<key>UIApplicationSupportsIndirectInputEvents</key>
 	<true/>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>fetch</string>
+		<string>processing</string>
```

Comment:
What does processing mean?

Reply:
A background task that may take minutes to complete.